components/Home/HomeDownloadButton.tsx

```diff
@@ -45,7 +46,9 @@ const HomeDownloadButton: FC<HomeDownloadButtonProps> = ({
           </LocalizedLink>
         </li>
         <li>
-          <LocalizedLink href={getNodejsChangelog(node)}>
+          <LocalizedLink
+            href={getNodejsChangelog(`v${nodeReleaseData.version}`)}
```

Comment:
Doesn't the nodeReleaseData already has a variable with the v prefix?

Reply:
Initially, they are:
`v20.1.0` and `v20.x`

Now:
`20.1.0` and `20`

IMO I prefer to let components taking care of the formatting. Because we need something like `20.1.0` as well.

Reply:
Since the v prefix is probably common, we can incorporate this on the NodeReleaseData as a `versionWithSuffix` or create a utility to add the prefix. It might be something silly. Or we can simply remove the `v` prefix from the argument for that function and update its logic to do whatever it needs without the `v` prefix or alternatives.

Reply:
I think it's better to have `versionWithPrefix` because it's actually very common, thanks!